x/paramHub/hub.go

```diff
@@ -83,6 +83,15 @@ func RegisterUpgradeBeginBlocker(paramHub *ParamHub) {
 		}
 		paramHub.UpdateFeeParams(ctx, crossStakeFeeParams)
 	})
+	sdk.UpgradeMgr.RegisterBeginBlocker(sdk.BEPHHH, func(ctx sdk.Context) {
+		updateFeeParams := []param.FeeParam{
+			&param.FixedFeeParams{MsgType: "edit_validator", Fee: EditSideChainValidatorFee, FeeFor: sdk.FeeForProposer},
```

Comment:
A bit weird we still use `EditSideChainValidatorFee`,  `Side` may be not proper, try define new one?

Reply:
Tried to reuse the original params. Indeed it's not that clear, I'll copy the params and rename them.

Reply:
Fixed